modules/application.js

```diff
@@ -0,0 +1,125 @@
+class Application {
+  constructor() {
+    this.contactForm = document.querySelector('.book_form');
+    this.title = document.querySelector('.book_title');
+    this.author = document.querySelector('.book_author');
+    this.bookList = document.querySelector('.parent_book_container');
+    this.dateTime = document.querySelector('.date_time');
+    this.bookListSection = document.querySelector('.book_list_section');
+    this.addBookSection = document.querySelector('.add_book_section');
+    this.contactSection = document.querySelector('.contact_section');
+
+    this.booksArray = [];
+    this.monthMap = ['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December'];
+
+    this.contactForm.addEventListener('submit', (event) => {
+      event.currentTarget.ref.addBook();
+      event.preventDefault();
+    });
+    this.contactForm.ref = this;
+
+    document.querySelector('.nav1').addEventListener('click', (event) => {
+      event.currentTarget.ref.displayBookListSection();
+    });
+    document.querySelector('.nav1').ref = this;
+
+    document.querySelector('.nav2').addEventListener('click', (event) => {
+      event.currentTarget.ref.displayAddBookSection();
+    });
+    document.querySelector('.nav2').ref = this;
+
+    document.querySelector('.nav3').addEventListener('click', (event) => {
+      event.currentTarget.ref.displayContactSection();
+    });
+    document.querySelector('.nav3').ref = this;
+
+    if (localStorage.getItem('books') != null) {
+      this.displayBookListSection();
+      this.booksArray = JSON.parse(localStorage.getItem('books'));
+      this.intitializeDocument();
+      this.intitializeRemoveButtonEvents();
+    }
+
+    window.setInterval(() => {
+      // eslint-disable-next-line no-undef
```

Comment:
- [OPTIONAL] `Linters` are tools that perform static analysis on software in order to recognize and report adherence/non-adherence to some set of coding best practice.Instead of disabling `eslint`, it will be nice to fix the issue.